src/distrs/beta.jl

```diff
@@ -1,7 +1,6 @@
 # functions related to beta distributions
 
 import .RFunctions:
-    betalogpdf,
```

Comment:
I start thinking it should be recognized as a bug of `ForwardDiff.jl` because the `Dual` type in their package inherits from `Real` but actually the true relation should be `Dual` extents `Real`.